crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -472,6 +449,26 @@ impl ViewEye {
         did_interact
     }
 
+    fn linear_speed_from_property(
+        &self,
+        bounding_boxes: &SceneBoundingBoxes,
+        view_ctx: &ViewContext<'_>,
+        eye_property: &ViewProperty,
+    ) -> f32 {
+        let eye_linear_speed = eye_property.component_or_fallback::<LinearSpeed>(
+            view_ctx,
+            self,
+            &Eye3D::descriptor_translation_speed(),
+        );
+        match eye_linear_speed {
+            Ok(linear_speed) => **linear_speed as f32,
+            Err(err) => {
+                re_log::error!("Error while getting linear speed for eye {}", err);
```

Comment:
as above, just as a precaution
```suggestion
                re_log::error_once!("Error while getting linear speed for eye {}", err);
```